src/box/lua/schema.lua

```diff
@@ -1600,6 +1601,7 @@ local alter_index_template = {
     type = 'string',
     parts = 'table',
     sequence = 'boolean, number, string, table',
+    filters = 'table',
```

Comment:
Nit. We don't need add filters explicitly here. The below loop will do.

Reply:
Thanks, fixed!